srcpkgs/llvm8/files/patches/cfe/cfe-004-add-musl-triples.patch

```diff
@@ -0,0 +1,57 @@
+--- clang/lib/Driver/ToolChains/Gnu.cpp
++++ clang/lib/Driver/ToolChains/Gnu.cpp
+@@ -1871,7 +1871,7 @@ void Generic_GCC::GCCInstallationDetector::AddDefaultGCCPrefixes(
+   static const char *const AArch64LibDirs[] = {"/lib64", "/lib"};
+   static const char *const AArch64Triples[] = {
+       "aarch64-none-linux-gnu", "aarch64-linux-gnu", "aarch64-redhat-linux",
+-      "aarch64-suse-linux", "aarch64-linux-android"};
++      "aarch64-suse-linux", "aarch64-linux-android", "aarch64-linux-musl"};
+   static const char *const AArch64beLibDirs[] = {"/lib"};
+   static const char *const AArch64beTriples[] = {"aarch64_be-none-linux-gnu",
+                                                  "aarch64_be-linux-gnu"};
+@@ -1882,7 +1882,9 @@ void Generic_GCC::GCCInstallationDetector::AddDefaultGCCPrefixes(
+   static const char *const ARMHFTriples[] = {"arm-linux-gnueabihf",
+                                              "armv7hl-redhat-linux-gnueabi",
+                                              "armv6hl-suse-linux-gnueabi",
+-                                             "armv7hl-suse-linux-gnueabi"};
++                                             "armv7hl-suse-linux-gnueabi",
++                                             "arm-linux-musleabihf",
++                                             "armv7l-linux-musleabihf"};
```

Comment:
to fix the linking problem on armv7 we need `armv7l-linux-gnueabihf` in this list

Reply:
Fixed, I can't seem to see any other missing triplets, or do you know any?

Reply:
Thank you. I thinks it's complete now.

Reply:
yes